python/cudf/cudf/utils/gpu.pxd

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
Should go into `_libxx` folder

Reply:
We'd want this to run before any of the rest of imports in `cudf.__init__.py` so placing this under `_libxx` and trying to import rest of code is not possible, that's why separated it out into utils folder.

Reply:
We should be able to have it in `_libxx` but not in the `__init__` of  `_libxx` though, no?

Reply:
Discussed offline and we decided to have a `_cuda` module under `cudf/python/cudf/_cuda`